docs/cloudformation.json

```diff
@@ -301,6 +301,7 @@
             {
               "Effect": "Allow",
               "Action": [
+                "cloudformation:*",
```

Comment:
When this is ready, need to determine exact permissions that Empire needs (and maybe we can lock this down so it can only access stacks that it created).